.circleci/config.yml

```diff
@@ -47,16 +47,20 @@ jobs:
         environment:
             COLUMNS=180
     steps:
+      - restore_cache:
+          keys: sample-data-v1
+
       - checkout
       - run: *apt-install
       - run: *docs-install
       # Copy and paste this line for every submodule for which the tests pass
       - run: conda install openjpeg
       - run: python -m "sunpy.data.sample"
       - run: python setup.py test -P time
-      - run: python setup.py test -P net --online
+      # - run: python setup.py test -P net --online
```

Comment:
I assume this was because VSO was timing out? It should be back now.

Reply:
That's because net takes almost half an hour to complete running.

Reply:
Why don't we put the online tests into a seperate `astropy-time-online` circle ci job?

Reply:
Yeah. Sounds good.

Reply:
Do you want to add that here and then we can merge this?

Reply:
Yeah. I'll add it now.